packages/app/src/app/pages/Sandbox/Editor/Header/ForkButton/ForkButton.tsx

```diff
@@ -153,64 +166,79 @@ export const ForkButton: React.FC<ForkButtonProps> = props => {
-                ? {}
-                : {
-                    width: '26px',
-                    borderTopLeftRadius: 0,
-                    borderBottomLeftRadius: 0,
-                  }
-            }
-          >
-            {state.activeWorkspaceAuthorization === 'READ' ? (
-              <>
-                <ForkIcon css={css({ height: 3, marginRight: 1 })} />{' '}
-                <Text css={css({ marginRight: 2 })}>Fork</Text>
-              </>
-            ) : null}
-            <Icon size={8} name="caret" />
-          </Menu.Button>
-          <Menu.List
-            css={css({
-              fontSize: 2,
-              lineHeight: 1,
-            })}
-            style={{
-              paddingTop: 4,
-              paddingBottom: 4,
-              marginLeft: '-4rem',
-              marginTop: -4,
-            }}
-          >
-            {currentSpace && (
-              <TeamOrUserItem
-                forkClicked={props.forkClicked}
-                item={currentSpace}
-                disabled={state.activeWorkspaceAuthorization === 'READ'}
-                isPersonal={currentSpace.teamId === state.personalWorkspaceId}
-              />
-            )}
-
-            <Menu.Divider />
-            {otherWorkspaces.map((space, i) => (
-              <TeamOrUserItem
-                isPersonal={space.teamId === state.personalWorkspaceId}
-                key={space.teamId}
-                forkClicked={props.forkClicked}
-                item={space}
-                disabled={
-                  space.userAuthorizations.find(
-                    authorization => authorization.userId === user.id
-                  )?.authorization === 'READ'
+        <Tooltip
+          label={
+            preventForksOutsideWorkspace
+              ? 'You don not have permission to fork this sandbox outside of the workspace'
+              : null
+          }
+        >
+          <div>
+            <Menu>
+              <Menu.Button
+                variant={props.variant}
+                disabled={preventForksOutsideWorkspace}
+                css={
+                  preventForkInsideWorkspace
+                    ? {}
+                    : {
+                        width: '26px',
+                        borderTopLeftRadius: 0,
+                        borderBottomLeftRadius: 0,
+                      }
                 }
-              />
-            ))}
-          </Menu.List>
-        </Menu>
+              >
+                {preventForkInsideWorkspace ? (
+                  <>
+                    <ForkIcon css={css({ height: 3, marginRight: 1 })} />{' '}
+                    <Text css={css({ marginRight: 2 })}>Fork</Text>
+                  </>
+                ) : null}
+
+                <Icon size={8} name="caret" />
+              </Menu.Button>
+
+              <Menu.List
+                css={css({
+                  fontSize: 2,
+                  lineHeight: 1,
+                })}
+                style={{
```

Comment:
Why is this not in the css?

Reply:
I'm not sure why was it not css earlier, I assumed Ives had a reason. Not going to lie, I nested it without trying to challenge what's inside.